CMakeLists.txt

```diff
@@ -193,6 +183,19 @@ nest_check_have_xlc_ice_on_using()
 nest_check_have_std_nan()
 nest_check_have_std_isnan()
 
+################################################################################
+##################           Create version string            ##################
+################################################################################
+
+include( NestVersionInfo )
+get_version_info()
+
+message("-- NEST version: ${NEST_VERSION_STRING}")
+#message("NEST_VERSION_BRANCH \"${NEST_VERSION_BRANCH}\"")
+#message("NEST_VERSION_SUFFIX \"${NEST_VERSION_SUFFIX}\"")
+#message("NEST_VERSION \"${NEST_VERSION}\"")
+#message("NEST_VERSION_GITHASH \"${NEST_VERSION_GITHASH}\"")
```

Comment:
Can these commented out lines be removed?

Reply:
We left them in as some kind of documentation. I have replaced them by proper documentation for all the flags in `NestVersionInfo.cmake` and for the added `cmake` options in `INSTALL` in 4e361c3. Thanks for being picky about this!

Reply:
Great!